docs/tutorials/porch-development-environment/README.md

```diff
@@ -375,11 +82,7 @@ Sample output:
 NAME                     SERVICE            AVAILABLE   AGE
 v1alpha1.porch.kpt.dev   porch-system/api   True        18m
 ```
-
-
 Check the porch api-resources:
-
-We should now also have the `porch.kpt.dev/v1alpha1` resources available
 ```
 kubectl api-resources | grep porch
```

Comment:
When the server is up the output should be:

functions                                      config.porch.kpt.dev/v1alpha1     true         Function
packagerevs                                    config.porch.kpt.dev/v1alpha1     true         PackageRev
packagevariants                                config.porch.kpt.dev/v1alpha1     true         PackageVariant
packagevariantsets                             config.porch.kpt.dev/v1alpha2     true         PackageVariantSet
repositories                                   config.porch.kpt.dev/v1alpha1     true         Repository
functions                                      porch.kpt.dev/v1alpha1            true         Function
packagerevisionresources                       porch.kpt.dev/v1alpha1            true         PackageRevisionResources
packagerevisions                               porch.kpt.dev/v1alpha1            true         PackageRevision
packages                                       porch.kpt.dev/v1alpha1            true         PorchPackage

Reply:
thank you. fixed.